openedx_lti_tool_plugin/templates/openedx_lti_tool_plugin/base.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang='en'>
```

Comment:
Can we use the Django setting value? @kuipumu In case other platforms have a different value.